tdms/include/simulation_parameters.h

```diff
@@ -114,4 +127,11 @@ class SimulationParameters{
     void set_spacing_stride(const double* vector);
```

Comment:
Weird. Did I not write a docstring!?

Reply:
😱 You've got to watch this guy, [he'll try to get away with schenanegans](https://github.com/UCL/TDMS/pull/200/files).